PaddleAudio/examples/dcase2021_task1b/requirements.txt

```diff
@@ -0,0 +1,4 @@
+paddleaudio
+# for clip linear-probe
+scikit-learn~=0.24.1
```

Comment:
Why not just build a simple FC network as classification instead of scikit.
It seems not that reasonable as PaddlePaddle ofiicial repo. It's ok as a third-party repo.

Reply:
For this video branch example, we use sk-learn. If users are not interested in  evaluate_visual_lp.py, the third-party sk-learn is not required.